packages/contracts/contracts/appDefinitions/HighRollerApp.sol

```diff
@@ -0,0 +1,179 @@
+pragma solidity 0.4.25;
+pragma experimental "ABIEncoderV2";
+
+import "../lib/Transfer.sol";
+
+/*
+  This contract is for a dice rolling game.
+  Two players roll two dice each.
+  The winner is the player whose sum of dice outcomes is highest.
+*/
```

Comment:
This docstring doesn't follow the natspec https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format